lib/tapi_tcp_states/tapi_tcp_states.c

```diff
@@ -1045,6 +1045,25 @@ get_tcp_move(rpc_tcp_state state_from, rpc_tcp_state state_to,
     return act;
 }
 
+/**
+ * There are some cases where tsa_do_tcp_move() fail if we trust getsockopt().
+ * Here we consider this bugged moves.
```

Comment:
s/bugged/special/

Please write an explanation in human language.  Something like:

There can't exist a socket SYN_RECV state, but we consider a listening socket with an embryonic connection in SYN-RECV sate as such.

TIME_WAIT sockets are reported by Linux as CLOSED.

(I'm not sure my explanation is correct, and this is why I'd like to read it from you.)